gcmt/utils/interface.py

```diff
@@ -398,25 +410,26 @@ def phase_curve(self, phases, spectra=None, filename=None):
             coords={'phase': phases,
                     'wlen': spectra.wlen})
 
-    def _get_stellar_spec(self, wlen, Tstar):
+    def _get_stellar_spec(self, wlen, t_star):
```

Comment:
again same thing

Reply:
As seen above: here we would not change it. Let me know if this would be ok for you